modules/library/render/src/main/java/org/geotools/renderer/lite/gridcoverage2d/GridCoverageRenderer.java

```diff
@@ -748,7 +743,18 @@ private GridCoverage2D renderCoverage(final GridCoverage2D gridCoverage,
         }
     }
 
-
+	private RenderedImage getImageFromParentCoverage(GridCoverage2D parentCoverage) {
+		if (parentCoverage == null) {
+			return null;
+		}
+		RenderedImage ri = parentCoverage.getRenderedImage();
+		if (ri != null) {
+			PlanarImage pi = PlanarImage.wrapRenderedImage(ri);
+			pi.setProperty("ParentCoverage", parentCoverage);
```

Comment:
Best to put this name in a public constant for clients and tests to reuse

Reply:
Right. I forgot to do that. Fixing...